api/src/main/resources/api/service_openapi.yaml

```diff
@@ -1158,6 +1158,19 @@ components:
           type: array
           items:
             type: string
+        includeHierarchyFields:
+          description: Hierarchy fields to include in the returned instances
+          type: object
+          properties:
+            hierarchies:
```

Comment:
I don't understand the model here. What is this an array of? Why doesn't the response also have an array?

Reply:
This is an array of hierarchy names. Intent is to handle entities with multiple hierarchies (e.g. source, standard).

You're right, I forgot to handle this in the response. Will update.

Reply:
Updated the response to also return an array of hierarchy field sets.

Reply:
Ah, I was thinking that might be the case but I could see how it worked. Makes sense now with the response changes.